civihr_default_theme/template.php

```diff
@@ -359,6 +360,59 @@ function _is_hrreports_current_path() {
 }
 
 /**
+ * Builds the markup of the cog menu
+ *
+ * @return string
+ */
+function _build_cog_menu_markup() {
+  $menuItems = _get_cog_menu_items();
+  $markup = "";
+
+  foreach($menuItems as $menuItem) {
+    if (isset($menuItem['separator']) && $menuItem['separator'] == TRUE) {
+      $markup .= "<hr class=\"chr_header__sub-menu__separator\">";
```

Comment:
` $markup .= '<hr class="chr_header__sub-menu__separator">';`
We can use single quote, so that we dont need to escape double quotes.